### PR TITLE
samples: wifi: shell: Fix regulatory domain setting reference

### PR DESCRIPTION
**Pull request by D-Triveni**
Correct the display of the `Set regulatory domain` command in the testing SAP mode procedure section of the shell sample documentation.

Fixes SHEL-2732

### CODE REVIEW

**Review by Harini37 — APPROVED** [review]

**Review by richabp — APPROVED** [review]

File: samples/wifi/shell/README.rst [hunk 1]
```diff
@@ -430,7 +430,7 @@ To test the SAP mode, the sample must be built using the configuration overlay :
 
    .. code-block:: console
 
-      wifi reg_domain IN
+      wifi reg_domain US
```
Comment (krish2718) [inline samples/wifi/shell/README.rst:433]: Why is this change needed? I intentionally kept it as IN, 5GHz should work with IN.

Comment (krish2718) [reply samples/wifi/shell/README.rst:433]: Commit log need to be reworded? What is `display`?

Comment (D-Triveni) [reply samples/wifi/shell/README.rst:433]: > Why is this change needed? I intentionally kept it as IN, 5GHz should work with IN.

There seems to be a mismatch in the documentation where it mentions setting the regulatory domain to `US` but provides the command for `IN`, creating confusion.

Comment (D-Triveni) [reply samples/wifi/shell/README.rst:433]: As you kept IN intentionally, will edit the line which is mentioning about how to set regualtory in the document.

Comment (D-Triveni) [reply samples/wifi/shell/README.rst]: addressed.

**Review by krish2718 — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Review by rado17 — APPROVED** [review]
